code_generator/End2End/schema/nested_typenames.schema

```diff
@@ -0,0 +1,26 @@
+package improbable.test_schema;
+
+// Uncomment this file when Nested Type definitions are implemented.
```

Comment:
if we leave TODOs in code, then please also add a TODO and the ticket number to this file

Reply:
Done